py3status/modules/xsel.py

```diff
@@ -22,8 +25,13 @@
 SAMPLE OUTPUT
 {'full_text': 'selected text'}
 
-example
-{'full_text': 'rrrrr > wtf is a warlock doing in here'}
+Example:
+xsel {
+  max_size = 50
+  command = "xsel --clipboard --output"
+  on_click 1 = "exec xsel --clear --clipboard"
+  log_file = "~/.local/share/xsel/clipboard_log"
+}
```

Comment:
@JohnAZoidberg 

One last thing can you move you Example above the `SAMPLE OUTPUT` line and replace the
```
example
{'full_text': 'rrrrr > wtf is a warlock doing in here'}
```

These are used for creating the documentation see https://py3status.readthedocs.io/en/3.6/modules.html#xsel

Reply:
`{'full_text': '<grrrrr > wtf is a warlock doing in here'}` pls

Reply:
What do you want to do with `{'full_text': '<grrrrr > wtf is a warlock doing in here'}`? Add it back in?
Does this really add value?

Reply:
>Does this really add value?

Yes. The images in that link comes from the modules themselves.

We generate the screenshots with that. It is a good real-time example for `xsel` too as you had this in your clipboard and probably in your log too. All modules have a `SAMPLE OUTPUT` strings.

Reply:
@JohnAZoidberg use your judgement the main thing is to move your example above the sample output